### PR TITLE
GSW-1402 Unstaking APR

### PR DESCRIPTION
**Pull request by AkydA**
Description
---
- Fix: unstaking apr calculation
- Refactor: external link constants
- Tooltip
  - optimize callback condition
  - disable auto hide scrollbar

### CODE REVIEW

File: packages/web/src/components/unstake/unstake-position-modal/UnstakePositionModal.tsx [hunk 1]
```diff
@@ -45,54 +47,47 @@ const UnstakePositionModal: React.FC<Props> = ({
...
-              allUsd: acc.allUsd + current.usdValue,
-            };
-          },
-          {
-            unstakeUsd: 0,
-            allUsd: 0,
-          },
-        ) ?? 0;
+  const {feeApr, totalApr} = useMemo(() => {
+    console.log(positions);
```
Comment (jinoosss) [inline packages/web/src/components/unstake/unstake-position-modal/UnstakePositionModal.tsx:51]: Is this the log you need?
If not, it's probably a better idea to remove it.

File: packages/web/src/components/unstake/unstake-position-modal/UnstakePositionModal.tsx [hunk 2]
```diff
@@ -45,54 +47,47 @@ const UnstakePositionModal: React.FC<Props> = ({
...
 
-    return formatRate(result.unstakeUsd / result.allUsd);
-  }, [positions]);
+          return accum;
+        },
+        { fee: 0, rewards: 0 },
+      );
+      return { liquidity: position.liquidity, aprs };
+    });
+    console.log(positionAprs);
```
Comment (jinoosss) [inline packages/web/src/components/unstake/unstake-position-modal/UnstakePositionModal.tsx:66]: Is this the log you need?
If not, it's probably a better idea to remove it.

File: packages/web/src/components/unstake/unstake-position-modal/UnstakePositionModal.tsx [hunk 3]
```diff
@@ -45,54 +47,47 @@ const UnstakePositionModal: React.FC<Props> = ({
...
+          currentPostionApr.liquidity;
+        accum.rewards +=
+          BigInt((currentPostionApr.aprs.rewards * 1000).toFixed(0)) *
+          currentPostionApr.liquidity;
+        accum.liquidity += currentPostionApr.liquidity;
+        return accum;
+      },
+      { fee: 0n, rewards: 0n, liquidity: 0n },
+    );
+    console.log(result);
```
Comment (jinoosss) [inline packages/web/src/components/unstake/unstake-position-modal/UnstakePositionModal.tsx:81]: Is this the log you need?
If not, it's probably a better idea to remove it.

**Review by jinoosss — COMMENTED** [review]
I left some minor comments.

**Review by jinoosss — APPROVED** [review]

**Review by notJoon — APPROVED** [review]

**Pull request merged**